cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/executor/ReplicationThrottleHelper.java

```diff
@@ -36,13 +37,21 @@
   static final String FOLLOWER_THROTTLED_RATE = "follower.replication.throttled.rate";
   static final String LEADER_THROTTLED_REPLICAS = LogConfig.LeaderReplicationThrottledReplicasProp();
   static final String FOLLOWER_THROTTLED_REPLICAS = LogConfig.FollowerReplicationThrottledReplicasProp();
+  static final int RETRIES = 30;
+  static final long RETRY_INTERVAL_MS = 1000L;
```

Comment:
Nit: Would it be possible to adopt retries with backoff? Please see `CruiseControlMetricsUtils#retry` for an example.

Reply:
Since this module already depends on cruise-control-metrics-reporter (where `CruiseControlMetricsUtils#retry` is defined), I reused the method.